Makefile

```diff
@@ -42,4 +42,7 @@ rpm-build/.exists:
 	mkdir -p rpm-build
 	touch rpm-build/.exists
 
-
+# For devel convinience
+install:
+	sudo rm -rf dist/ build/ 
```

Comment:
Agreed. But since it is just for developer's convenience, let us leave it as it is for now.